public/diagnostic.js

```diff
@@ -23,3 +23,24 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+
+  var orders: Ember.A(),
+
+  addToCart: function(item) {
+    this.get('order').addObject(item);
+  },
+
+  totalPrice: Ember.computed.sum('orders.@each.orderPrice'),
```

Comment:
You're tracking the right key, but this is not a correct use of `sum`. `sum` will try to all all of the elements in the array, and this array is an array of Orders, not numbers. You could create an intermediary computed property using `map` or `mapBy`, and then have another computed property to sum across that, or you could just create a custom computed property to both map and sum.